pulsar-broker/src/test/java/org/apache/pulsar/broker/service/PersistentTopicTest.java

```diff
@@ -2278,7 +2279,7 @@ public void testGetReplicationClusters() throws Exception {
         topicPolicies.setReplicationClusters(topicClusters);
         Optional<TopicPolicies> optionalTopicPolicies = Optional.of(topicPolicies);
         topicPoliciesFuture.complete(optionalTopicPolicies);
-        when(topicPoliciesService.getTopicPoliciesAsyncWithRetry(any(), any(), any())).thenReturn(topicPoliciesFuture);
+        when(topicPoliciesService.getTopicPoliciesAsyncWithRetry(any(), any(), any(), anyBoolean())).thenReturn(topicPoliciesFuture);
```

Comment:
What about expecting an exact value for the boolean parameter and not anyBoolean ?

Reply:
Whether it is true or false, mock is required here, otherwise there will be NPE.